configure.ac

```diff
@@ -1,8 +1,8 @@
 AC_INIT(system-config-printer, 1.5.15)
 AC_CONFIG_SRCDIR(system-config-printer.py)
 AM_INIT_AUTOMAKE([dist-xz dist-bzip2 subdir-objects 1.6])
-IT_PROG_INTLTOOL
-AM_GNU_GETTEXT_VERSION([0.18.3])
+AM_GNU_GETTEXT_VERSION(0.20)
+AX_REQUIRE_DEFINED([AM_GNU_GETTEXT])
```

Comment:
@inigomartinez AX_REQUIRES_DEFINED() macro requires ```autoconf-archives``` package, would you mind adding it into build requirements in README.md?